org.opent2t.sample.lamp.superpopular/com.insteon.lightbulb/js/README.md

```diff
@@ -0,0 +1,43 @@
+#Insteon Light Translator
+Translator forInsteon Light (http://www.insteon.com)
```

Comment:
#Resolved

---
In reply to: [82906057](https://github.com/openT2T/translators/pull/107#discussion_r82906057) [](ancestors = 82906057)